packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -115,7 +115,7 @@ const GlobalStylesVariations = ( {
 
 	const variationDescription = needsUpgrade
 		? translate(
-				'Unlock style variations and tons of other features with the %(planName)s plan, or try them out now for free.',
+				'Preview our style variations for free or pick your own fonts and colors with the %(planName)s plan later on.',
 				{ args: { planName: getPlan( upgradeToPlan )?.getTitle() ?? '' } }
 		  )
 		: translate( 'You can change your style at any time.' );
```

Comment:
Since we're using the fallback here, can we update it to its previous text, `Unlock style variations and tons of other features with the %(planName)s plan, or try them out now for free.`? 

Or, if we plan to keep the PR open until the translations arrive, can we remove this fallback entirely?

Reply:
Yeah I'm just string freeze and remove the fallback altogether